src/cryptonote_basic/cryptonote_basic_impl.cpp

```diff
@@ -106,11 +108,8 @@ namespace cryptonote {
       base_reward = 0;
     }
 
-    uint64_t minimum_reward = already_generated_coins / (720 * 365 * YEARLY_INFLATION_INVERSE);
-    if (base_reward < minimum_reward)
-    {
-      base_reward = minimum_reward;
-    }
+    if (version >= 8)
+      base_reward = 28000000000 + 100000000000 / pow((double)2, height / (double)(720 * 90)); // halve every 90 days.
```

Comment:
pow (2.0, height / (720.0 / 90.0)) would be nice but not necessary.

Reply:
![](https://media.giphy.com/media/2vs93F78Xoyn68EQ9k/giphy.gif)